app/views/trips/index.erb

```diff
@@ -0,0 +1,42 @@
+<table class="table">
+  <thead>
+    <tr>
+      <th>Trip ID</th>
+      <th>Duration (in seconds)</th>
+      <th>Start Date</th>
+      <th>Start Station ID</th>
+      <th>End Date</th>
+      <th>End Station Id</th>
+      <th>Bike ID</th>
+      <th>Subscription Type</th>
+      <th>Zip Code</th>
+      <th>Delete</th>
+      <th>Update</th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <!-- <% count = 0 %> -->
+    <th scope="row" id="total entries"></th>
+    <%  @trips.each do |trip| %>
+    <tr>
+      <td><%= trip.id %></td>
+      <td><%= trip.duration_in_seconds%></td>
+      <td><%= trip.start_date%></td>
+      <td><%= @stations.find_by(id: trip.start_station_id).name%></td>
```

Comment:
this is an example of querying your database in your views since you're calling `find_by` - how can we remove this logic and keep our views simply displaying data that the controller fetches for us?